Resources/Prototypes/_SpaceCats/Entities/Objects/Specific/Research/Xenobiology/potions.yml

```
@@ -0,0 +1,110 @@
+  - type: Sprite
+    sprite: _SpaceCats/Objects/Specific/Xenobiology/potion.rsi
+    state: orange
+
+- type: entity
+  parent: BaseSlimePotion
+  id: PurpleSlimePotion
+  name: purple slime potion
+  components:
+  - type: Sprite
+    sprite: _SpaceCats/Objects/Specific/Xenobiology/potion.rsi
+    state: purple
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)